tcms/testruns/tests/test_views.py

```diff
@@ -122,6 +122,30 @@ def test_show_create_new_run_page(self):
                 '<a href="%s">TC-%d: %s</a>' % (case_url, case.pk, case.summary),
                 html=True)
 
+    def test_should_be_able_to_create_new_run(self):
+        new_run_summary = 'TestRun summary'
+        post_data = {
+            'summary': new_run_summary,
+            'plan': self.plan.pk,
+            'product_id': self.plan.product_id,
+            'product': self.plan.product_id,
+            'product_version': self.plan.product_version.pk,
+            'build': self.build.pk,
+            'manager': self.tester.email,
+            'default_tester': self.tester.email,
+            'notes': '',
+            'case': [self.case_1.pk, self.case_2.pk]
+        }
+
+        url = reverse('testruns-new')
+        response = self.client.post(url, post_data)
+
+        new_run = TestRun.objects.get(summary=new_run_summary)
+
+        self.assertRedirects(
+            response,
+            reverse('testruns-get', args=[new_run.pk]))
+
```

Comment:
Can also iterate(filter) executions of the new TR and make sure they match case_1 and case_2 above. A .filter().exists() on the m2m field in TestRun should do the trick.